app.rb

```diff
@@ -0,0 +1,150 @@
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require 'colorize'
+
+class App
+  def initialize
+    @books = []
```

Comment:
- [ ] Great job 💪 . Unfortunately, your app doesn't work when I try running it in the console. Kindly fix this error 🙏 

![image](https://user-images.githubusercontent.com/74814780/201487828-252ba3c3-a361-416d-bd42-dbaaf6348ff0.png)